src/main/scripts/brew-update-to-final

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh -ex
```

Comment:
Maybe we should also add `-u` since we're relying on environment variables being set.